sound/soc/sof/ipc4.c

```diff
@@ -597,11 +597,21 @@ static void sof_ipc4_rx_msg(struct snd_sof_dev *sdev)
 	}
 }
 
+static int sof_ipc4_ctx_save(struct snd_sof_dev *sdev)
```

Comment:
why do you say "as an example" in the commit message? It is exactly how it is done isnt it?